aries-site/src/pages/components/menu.js

```diff
@@ -22,22 +28,147 @@ const Menu = () => {
         <Subsection name={title} level={1} topic={topic}>
           <SubsectionText>{page.description}</SubsectionText>
           <Example
-            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/controls/MenuExample.js"
-            details={[
-              <Text size="large">
-                Menu is used to filter or sort content on a page. It is, in some
-                ways, similar to the{' '}
-                <Anchor href="input#select">Select component</Anchor>. However,
-                use the select component when the user must specify from a list
-                of options and submit.
-              </Text>,
-            ]}
+            code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/menu/MenuExample.js"
+            designer="https://designer.grommet.io/menu?id=HPE-design-system-hpedesignsystem-hpe-com"
             docs="https://v2.grommet.io/menu?theme=hpe#props"
+            figma="https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component"
           >
             <MenuExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            A Menu is a set of actions that may be used to engage with content
+            on a page or to navigate to other pages. When a menu item is
+            clicked, the action will be executed. Menu is distinct from{' '}
+            <Anchor href={nameToPath('Select')}>Select</Anchor>, which is often
+            used in the context of a Form to choose from a list of values but
+            does not perform actions.
+          </SubsectionText>
```

Comment:
```suggestion
          <SubsectionText>
            A Menu is a set of actions that may be used to engage with content
            on a page or to navigate to other pages. When a menu item is
            clicked, the action will be executed.
          </SubsectionText>
          <SubsectionText> 
            Menu should not be confused with{' '}
            <Anchor href={nameToPath('Select')}>Select</Anchor>. Though
            visually similar, Select is often used in the context of a Form and 
            used to choose from a set of values rather than actions.
          </SubsectionText>
```